### DISCUSSION TITLE
Implemented LeNet

### DISCUSSION THREAD

**Discussion by aitor-martinez-seras** [megvii-research/FSSD_OoD_Detection#4] [Q&A] [answered]
Hello,

I'm wondering why the LeNet implemented in this repository has not a MaxPooling2d layer between the first and the second convolution.  As far as I know, LeNet has this pooling layer after each convolution, and in the file lenet.py the MaxPooling is only applied after the second convolution.  Is this a mistake or an intented feature of the LeNet you have implemented?

Thank you very much for the repository, it is being very helpful

**Comment by andrehuang** [comment, marked as answer]
Hi,

It's great to hear that our repo is helpful to you!

As for the LeNet, we do recognize that there are other implementations of LeNet. When we choose the implementation, we chose to align with the Pytorch official [example repo](https://github.com/pytorch/examples/blob/master/mnist/main.py). So we adopted the LeNet version they used. 

It should not make a big difference if you change the architecture. But if it does, please let us know.
